CHANGELOG.md

```diff
@@ -66,20 +56,6 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 - Add support for the domain user attribute [(#127)](https://github.com/ManageIQ/manageiq-appliance/pull/127)
 - ExecStart requires an absolute executable path [(#151)](https://github.com/ManageIQ/manageiq-appliance/pull/151)
 
-## Unreleased as of Sprint 77 ending 2018-01-15
```

Comment:
Why are these deleted?  They are in this branch, right?

Reply:
They weren't deleted,...just moved from Unreleased state to the release.